package.json

```diff
@@ -2,21 +2,33 @@
   "name": "psi-cardinality",
   "version": "0.1.0",
   "description": "PSI cardinality for JS and WASM using Bazel and Emscripten",
-  "main": "psi_cardinality/javascript/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/OpenMined/psi-cardinality"
+  },
   "license": "Apache-2.0",
+  "keywords": [
+    "psi",
+    "private set intersection",
+    "private join and compute"
+  ],
+  "main": "psi_cardinality/javascript/index.js",
   "scripts": {
     "benchmark:js": "RUN_DEMO=js node -r esm psi_cardinality/javascript/benchmark.js",
     "benchmark:wasm": "RUN_DEMO=wasm node -r esm psi_cardinality/javascript/benchmark.js",
     "build": "yarn build:client && yarn build:server",
     "build:client": "yarn build:client:js && yarn build:client:wasm",
-    "build:client:js": "bash psi_cardinality/javascript/scripts/build-client-js.sh",
-    "build:client:wasm": "bash psi_cardinality/javascript/scripts/build-client-wasm.sh",
+    "build:client:js": "bash psi_cardinality/javascript/scripts/build-client-js.sh && yarn build:copy",
+    "build:client:wasm": "bash psi_cardinality/javascript/scripts/build-client-wasm.sh && yarn build:copy",
     "build:server": "yarn build:server:js && yarn build:server:wasm",
-    "build:server:js": "bash psi_cardinality/javascript/scripts/build-server-js.sh",
-    "build:server:wasm": "bash psi_cardinality/javascript/scripts/build-server-wasm.sh",
+    "build:server:js": "bash psi_cardinality/javascript/scripts/build-server-js.sh && yarn build:copy",
+    "build:server:wasm": "bash psi_cardinality/javascript/scripts/build-server-wasm.sh && yarn build:copy",
     "build:benchmark": "yarn build:benchmark:js && yarn build:benchmark:wasm",
-    "build:benchmark:js": "bash psi_cardinality/javascript/scripts/build-benchmark-js.sh",
-    "build:benchmark:wasm": "bash psi_cardinality/javascript/scripts/build-benchmark-wasm.sh",
+    "build:benchmark:js": "bash psi_cardinality/javascript/scripts/build-benchmark-js.sh && yarn build:copy",
+    "build:benchmark:wasm": "bash psi_cardinality/javascript/scripts/build-benchmark-wasm.sh && yarn build:copy",
+    "build:copy": "bash psi_cardinality/javascript/scripts/build-copy.sh",
```

Comment:
We should have a line like this in this script or put a `.gitkeep` file in the `bin` directory:
```shell
mkdir -p psi_cardinality/javascript/bin/
```

Reply:
fixed